units/src/amount/tests.rs

```diff
@@ -1196,184 +1196,133 @@ fn check_const() {
     assert_eq!(Amount::MAX_MONEY.to_sat() as i64, SignedAmount::MAX_MONEY.to_sat());
 }
 
-// Verify we have implemented all combinations of ops for `Amount` and `SignedAmount`.
-// It's easier to read this test that check the code.
+// Sanity check than stdlib supports the set of reference combinations for the ops we want.
```

Comment:
```suggestion
// Sanity check that stdlib supports the set of reference combinations for the ops we want.
```

Reply:
I realize this type existed before

Reply:
Tracked here to save force pushing: https://github.com/rust-bitcoin/rust-bitcoin/issues/1490#issuecomment-2683533748

Reply:
s/type/typo